test/common/upstream/eds_test.cc

```diff
@@ -247,6 +247,23 @@ TEST_F(EdsTest, OnConfigUpdateSuccess) {
   EXPECT_EQ(1UL, stats_.counter("cluster.name.update_no_rebuild").value());
 }
 
+// Validate that delta-style onConfigUpdate() with the expected cluster accepts config.
```

Comment:
Do we have tests for SDS and RDS delta updates?

Reply:
Added.